libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -6,6 +6,10 @@
 #include "PosVelEKF.h"
 #include <AP_HAL/utility/RingBuffer.h>
 
+#define EKF_INIT_TIME_MS 2000
```

Comment:
let's put a comment to the right of this declaration, "// EKF initialisation requries this many milliseconds of good sensor data"